wire-grpc-sample/build.gradle

```diff
@@ -2,31 +2,30 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 buildscript {
   ext.versions = [
-      'kotlin': '1.3.41',
-      'wire': '3.0.0',
-      'ktor': '1.2.3',
+      'kotlin': '1.3.71',
+      'wire': '3.2.1',
       'tcnative': '2.0.25.Final',
   ]
   ext.deps = [
       'agp': 'com.android.tools.build:gradle:3.5.0',
       'kotlin': [
           'plugin': "org.jetbrains.kotlin:kotlin-gradle-plugin:${versions.kotlin}",
           'stdlib': "org.jetbrains.kotlin:kotlin-stdlib-jdk8:${versions.kotlin}",
-          'coroutines': "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.0-RC",
-          'coroutinesAndroid': "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.0-RC",
+          'coroutines': "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.5",
+          'coroutinesAndroid': "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.5",
       ],
       'androidx': [
           'appcompat': "androidx.appcompat:appcompat:1.0.2",
           'constraintLayout': "androidx.constraintlayout:constraintlayout:1.1.3",
           'ktx': 'androidx.core:core-ktx:1.0.2',
       ],
       'wire': [
-          'runtime': "com.squareup.wire:wire-runtime-multiplatform:${versions.wire}",
+          'runtime': "com.squareup.wire:wire-runtime:${versions.wire}",
```

Comment:
FWIW `wire-runtime-multiplatform` and `wire-runtime` both work here, although `wire-runtime` needs less resolution steps.

Reply:
So the project would NOT compile with `wire-runtime-multiplatform` by the way.

Reply:
None of the Wire types would get resolved
![image](https://user-images.githubusercontent.com/1767669/80978351-eebaa480-8df3-11ea-872e-eacc9df61de3.png)

Reply:
This is strange, I would expect Gradle to resolve the dependencies properly.